Detectors/EMCAL/base/src/ClusterFactory.cxx

```diff
@@ -102,6 +103,21 @@ o2::emcal::AnalysisCluster ClusterFactory<InputType>::buildCluster(int clusterIn
   evalCoreEnergy(inputsIndices, clusterAnalysis);
   evalTime(inputsIndices, clusterAnalysis);
 
+  // set amp fractions
```

Comment:
this part is unrelated to the cell changes but still made it into this commit. Still useful addition though :-)